dashboard/lib/services/csta_enrollment.rb

```diff
@@ -61,4 +61,8 @@ def self.submit(first_name:, last_name:, email:, school_district_name:,
 
     nil
   end
+
+  def self.titleize_address(address)
+    address.titleize.gsub(/\b(N|S|E|W|NE|SE|NW|SW)\b/i, &:upcase)
```

Comment:
I think I understand the regex pattern here, but what does the `/i` do at the end?

Reply:
It's the [ignore case flag](https://ruby-doc.org/core-2.4.0/Regexp.html#class-Regexp-label-Options).  This pattern will match `ne` or `Ne` or `nE` in addition to `NE`.